tests/publiccloud/boottime.pm

```diff
@@ -346,12 +346,13 @@ sub store_in_db {
         os_flavor         => get_required_var('FLAVOR'),
         os_version        => get_required_var('VERSION'),
         os_build          => get_required_var('BUILD'),
-        os_pc_build       => get_required_var('PUBLIC_CLOUD_BUILD'),
-        os_pc_kiwi_build  => get_required_var('PUBLIC_CLOUD_BUILD_KIWI'),
         os_kernel_release => $results->{kernel_release},
         os_kernel_version => $results->{kernel_version},
     };
 
+    $tags{os_pc_build} = get_var('PUBLIC_CLOUD_QAM') ? 'N/A' : get_required_var('PUBLIC_CLOUD_BUILD');
```

Comment:
While this works, it appears a bit bulky to me. What do you think about the following instead?

```suggestion
    os_pc_build => get_var('PUBLIC_CLOUD_BUILD', 'N/A'),
    os_pc_kiwi_build  => get_var('PUBLIC_CLOUD_BUILD_KIWI', 'N/A'),
```

Reply:
Of course this goes where the original `get_required_var` was before, not in line 353.

Reply:
I would go with:
```suggestion
    if (!get_var('PUBLIC_CLOUD_QAM')) {
        $tags{os_pc_build} = get_required_var('PUBLIC_CLOUD_BUILD');
        ...
    }

```

Reply:
@grisu48  your version will not fail when Product QA run will miss this variables

Reply:
@cfconrad  your version is better but I am not sure what we win if we leave this field undefined ? 
My point here that someone who not aware with implementation ( or even me or you after 2 years of not touching this area :) ) might think that something is broken when she/he will see that part of flavors getting `PUBLIC_CLOUD_KIWI` / `PUBLIC_CLOUD_BUILD` variables

Reply:
maybe `N/A` is not best option we can choose anything else important part to put **something** there because for "future me" this will be the  message  "yes test is still aware about this column but just not have appropriate value for it" . While "undefined" ( or whatever influxdb will insert there ) will be another message for future me -  "hm maybe something is broken and we lost PUBLIC_CLOUD_KIWI somehwere on the way ?"